r2gg/__init__.py

```diff
@@ -1,7 +1,7 @@
 from r2gg._configure import configure
 from r2gg._main import sql_convert, pgr_convert, osm_convert, osrm_convert, valhalla_convert, write_road2_config
 
-__version__ = "1.2.4-DEVELOP"
+__version__ = "2.0.0"
```

Comment:
Can you keep the "develop" flag until we merge on master ? :) this is how we work on this project and its friends.

Reply:
I would rather keep the version without the develop flag since it will be used in a tag.